src/components/Error404.tsx

```diff
@@ -0,0 +1,27 @@
+import Menu from './Menu';
+
+let Error404 = () => {
```

Comment:
I'd do `const Error404` since this component should be constant.

Reply:
Would also suggest giving it the type `ReactElement`!